src/components/Attachment/FileAttachment.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import prettybytes from 'pretty-bytes';
+// @ts-expect-error
+import { FileIcon } from 'react-file-utils';
+
+import { SafeAnchor } from '../SafeAnchor';
+
+import type { Attachment } from 'stream-chat';
+
+import type { DefaultAttachmentType, UnknownType } from '../../../types/types';
+
+export type FileAttachmentProps<
+  At extends UnknownType = DefaultAttachmentType
+> = {
+  attachment: Attachment<At> & { file_size?: number };
```

Comment:
let's add { file_size?: number } to our DefaultAttachmentType and extend the default case here